lib/extractor/docker-archive/index.ts

```diff
@@ -15,7 +16,13 @@ export function getImageIdFromManifest(
   manifest: DockerArchiveManifest,
 ): string {
   try {
-    return manifest.Config.split(".")[0];
+    const imageId = manifest.Config.split(".")[0];
+    const hashAlgorithmPrefix = `${HashAlgorithm.Sha256}:`;
+    if (imageId.includes(":")) {
+      // imageId includes the algorithm prefix
+      return imageId;
+    }
+    return `${hashAlgorithmPrefix}${imageId}`;
```

Comment:
I'd separate the prefix from the delimiter, so we'll have `${hashAlgorithm}:{imageId}` instead of including the `:`